transports/sql/transactions_ambient-core-warning_sqltransport_[4,).partial.md

```diff
@@ -0,0 +1 @@
+WARNING: The Transaction scope mode is not available in .NET Core 2.0 because the implementation of `SqlConnection` does not support enlisting in an ambient transaction.
```

Comment:
This https://github.com/dotnet/corefx/issues/22484#issuecomment-316829947 is the only "documentation" I found.